packages/ketcher-react/src/script/ui/state/hotkeys.ts

```diff
@@ -72,6 +73,14 @@ function keyHandle(dispatch, state, hotKeys, event) {
       event.preventDefault()
       return
     }
+    // Removing from what should be saved - structure, which was added to paste tool,
```

Comment:
Propose to move this to separate function

Reply:
Updated, please check